modules/ROOT/assets/attachments/kubernetes/sgw-config-import.json

```
@@ -0,0 +1,44 @@
+{
+  "logging": {
+    "console": {
+      "log_file_path": "/var/tmp/sglogs",
+      "console": {
```

@jamiltz  Just realized that we should probably set console logging off by default
```suggestion
      "console": {
         "enabled":false
       }
```

---

https://github.com/couchbase/docs-sync-gateway/commit/c954f03659c5eb8e80792e9d74fce83e8646f021